packages/fast-jss-manager/src/index.ts

```diff
@@ -1,6 +1,12 @@
 import * as CSS from "csstype";
 import { ClassNames, IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
 export { ClassNames, IManagedClasses };
+
+/**
+ * Define a static CSS Rule
+ */
+export type ICSSStaticRule =  CSS.Properties | string;

```

Comment:
Suggest using @params for all type, functions, etc as well like what was done below.  Good practice to declare in comments the inputs and outputs to fully understand how it's used.